Bottom: 2529a03f98e608942ba83914abcab2768ebf6039
Top:    07b95bdb4669544ad757edccb3e200f792d6a569
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-25 02:21:15 +0100

Refresh of play-with-glsl-shader-for-rend

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 9e2221b..9ae6595 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2146,10 +2146,11 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   ghid_show_crosshair (TRUE);
 
   hidgl_flush_triangles (&buffer);
-  hidgl_finish_triangle_array (&buffer);
 
   draw_lead_user (priv);
 
+  hidgl_finish_triangle_array (&buffer);
+
   check_gl_drawing_ok_hack = false;
   hidgl_in_context (false);
   ghid_end_drawing (port);
